### PR TITLE
docs(cmd): updating cmd documentation

### PR DESCRIPTION
**Pull request by ramfox**
While writing the [RFC CLI commands](qri-io/rfcs#14) document, I figured I would go through and update the documentation on the CLI commands, if any are outdated.

### CODE REVIEW

File: cmd/registry.go [hunk 1]
```diff
@@ -44,36 +40,44 @@ $ qri config set registry.location ""`,
...
 	publish := &cobra.Command{
 		Use:   "publish",
-		Short: "publish dataset info to the registry",
+		Short: "Publish dataset info to the registry",
+		Long: `
+Publishes the dataset information onto the registry. There will be a record
+of your dataset on the registry, and if your dataset is less than 20mbs, 
+Qri will back your dataset up onto the registry.
+
+Published datasets can be found by other peers using the ` + "`qri search`" + ` command.
```
Comment (b5) [inline cmd/registry.go]: single quotes

File: cmd/render.go [hunk 2]
```diff
@@ -15,7 +15,15 @@ func NewRenderCommand(f Factory, ioStreams IOStreams) *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "render",
 		Short: "execute a template against a dataset",
-		Long:  `the most common use for render is to generate html from a qri dataset`,
+		Long: `
+You can use html templates, formatted in the go/html template style, 
+to render visualizations from your dataset. These visualizations can be charts, 
+graphs, or just display your dataset in a different format.
+
+Use the ` + "`--output`" + ` flag to save the rendered html to a file.
```
Comment (b5) [inline cmd/render.go]: single quotes

File: cmd/render.go [hunk 3]
```diff
@@ -15,7 +15,15 @@ func NewRenderCommand(f Factory, ioStreams IOStreams) *cobra.Command {
...
+
+Use the ` + "`--template`" + ` flag to use a custom template. If no template is
```
Comment (b5) [inline cmd/render.go]: single quotes

File: cmd/save.go [hunk 4]
```diff
@@ -24,20 +24,24 @@ func NewSaveCommand(f Factory, ioStreams IOStreams) *cobra.Command {
...
+Save is how you change a dataset, updating one or more of data, metadata, and structure. 
+You can also update your data via url. Every time you run save, an entry is added to 
+your dataset’s log (which you can see by running ` + "`qri log <dataset_reference>`" + `). 
+
+Every time you save, you can provide a message about what 
+you changed and why. If you don’t provide a message 
+Qri will automatically generate one for you.
+
+When you make an update and save a dataset that you originally added from a different
+peer, the dataset gets renamed from ` + "`peers_name/dataset_name`" + ` to ` + "`my_name/dataset_name`" + `.
```
Comment (b5) [inline cmd/save.go]: single quotes

File: cmd/setup.go [hunk 5]
```diff
@@ -21,17 +21,20 @@ func NewSetupCommand(f Factory, ioStreams IOStreams) *cobra.Command {
...
 - create an IPFS repository if one doesn’t exist
 
-This command is automatically run if you invoke any qri command without first 
-running setup. If setup has already been run, by default qri won’t let you 
-overwrite this info.`,
+This command is automatically run if you invoke any Qri command without first 
+running setup. If setup has already been run, by default Qri won’t let you 
+overwrite this info.
+
+Use the ` + "`--remove`" + ` to remove your Qri repo. This deletes your entire repo,
```
Comment (b5) [inline cmd/setup.go]: single quotes

File: cmd/validate.go [hunk 6]
```diff
@@ -48,20 +48,26 @@ schema for dataset "me/foo"
 Using validate this way is a great way to see how changes to data or schema
 will affect a dataset before saving changes to a dataset.
 
+You can get the current schema of a dataset by running the ` + "`qri get structure.schema`" + `
```
Comment (b5) [inline cmd/validate.go]: single quotes

**Comment by ramfox** [comment]
Okay before I go and remove all the back ticks, the reason I changed it in the first place is if we auto generate docs from this using cobra, then we won't have to go through and change all the single quotes to back ticks for the markdown files.

**Comment by b5** [comment]
Ooooooh ok fantastic. Def don't remove then. I think it'd be great if we had a `make cli-docs` command that generated the docs in question. We should also add a `Package cmd` note in `cmd/cmd.go` that explains why the backticks so others don't yell at us later :)

**Comment by ramfox** [comment]
@b5 
Added package notes, and added a `docs.go` file and script to the Makefile

a lil bit down and dirty, but didn't want to dump too much time into it.

**Pull request merged**